spec/abilities/admin_set_ability_spec.rb

```diff
@@ -1,178 +1,374 @@
 # frozen_string_literal: true
 require 'cancan/matchers'
 
-RSpec.describe Hyrax::Ability do
+RSpec.describe Hyrax::Ability, :clean_repo do
   subject { ability }
 
   let(:ability) { Ability.new(current_user) }
-  let(:user) { create(:user) }
+  let(:user) { create(:user, email: 'user@example.com') }
   let(:current_user) { user }
-  let(:admin_set) { create(:admin_set, edit_users: [user], with_permission_template: true) }
 
-  context 'when user who created the admin set' do
```

Comment:
This test condition was removed because it is really just testing MANAGE access due to the way the factories setup the creating user.